### PR TITLE
[DOCS] 6.8.4 release notes

### PR DESCRIPTION
**Pull request by tomcallahan**
:)

### CODE REVIEW

File: docs/reference/release-notes/6.8.asciidoc [hunk 1]
```diff
@@ -3,6 +3,79 @@
...
+* Always rebuild checkpoint tracker for old indices {pull}46340[#46340] (issues: {issue}38879[#38879], {issue}46311[#46311])
+
+Features/Indices APIs::
+* Fix Rollover error when alias has closed indices {pull}47839[#47839] (issue: {issue}47148[#47148])
+
+Features/Ingest::
+* Allow dropping documents with auto-generated ID {pull}46773[#46773] (issue: {issue}46678[#46678])
+
+Features/Java High Level REST Client::
+* Added fields for MultiTermVectors (#42232) {pull}42877[#42877] (issue: {issue}42232[#42232])
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc]: It looks like this was closed, but not merged.

File: docs/reference/release-notes/6.8.asciidoc [hunk 2]
```diff
@@ -3,6 +3,79 @@
 
 coming[6.8.4]
```
Comment (jrodewig) [inline docs/reference/release-notes/6.8.asciidoc]: We can remove this now.

```suggestion
```

**Review by jrodewig — CHANGES_REQUESTED** [review]
Looks good overall. Left a comment about a PR that needs to be removed before merge.

Thanks!

**Review by jrodewig — APPROVED** [review]
LGTM. Thanks!

**Comment by tomcallahan** [comment]
@elasticmachine run elasticsearch-ci/docs-check

**Pull request merged**